introduction.md

```diff
@@ -0,0 +1 @@
+Hey! My name is Ricardo. I just dropped my phd course, so I'm back to the coding game!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.